sensor/OldDataCleanSensor.js

```diff
@@ -197,104 +194,90 @@ class OldDataCleanSensor extends Sensor {
-      
-      let toBeDeleted = []
+    let toBeDeleted = []
 
-      for(let i = 0; i < policies.length; i++) {
-        let p = policies[i]
-        for(let j = i+1; j< policies.length; j++) {
-          let p2 = policies[j]
-          if(p && p2 && p.isEqualToPolicy(p2)) {
-            toBeDeleted.push(p)
-            break
-          }
+    for (let i = 0; i < policies.length; i++) {
+      let p = policies[i]
+      for (let j = i + 1; j < policies.length; j++) {
+        let p2 = policies[j]
+        if (p && p2 && p.isEqualToPolicy(p2)) {
+          toBeDeleted.push(p)
+          break
         }
       }
+    }
 
-      for(let k in toBeDeleted) {
-        let p = toBeDeleted[k]
-        await (pm2.deletePolicy(p.pid))
-      }
-    })()
+    for (let k in toBeDeleted) {
+      let p = toBeDeleted[k]
+      await pm2.deletePolicy(p.pid);
+    }
   }
 
-  cleanDuplicatedException() {
-    return async(() => {
-      let exceptions = [];
-      try {
-        exceptions = await(em.loadExceptionsAsync());
-      } catch (err) {
-        log.error("Error when loadExceptions", err);
-      }
+  async cleanDuplicatedException() {
+    let exceptions = [];
+    try {
+      exceptions = await em.loadExceptionsAsync();
+    } catch (err) {
+      log.error("Error when loadExceptions", err);
+    }
 
-      let toBeDeleted = []
+    let toBeDeleted = []
 
-      for(let i = 0; i < exceptions.length; i++) {
-        let e = exceptions[i]
-        for(let j = i+1; j< exceptions.length; j++) {
-          let e2 = exceptions[j]
-          if(e && e2 && e.isEqualToException(e2)) {
-            toBeDeleted.push(e)
-            break
-          }
+    for (let i = 0; i < exceptions.length; i++) {
+      let e = exceptions[i]
+      for (let j = i + 1; j < exceptions.length; j++) {
+        let e2 = exceptions[j]
+        if (e && e2 && e.isEqualToException(e2)) {
+          toBeDeleted.push(e)
+          break
         }
       }
+    }
 
-      for(let k in toBeDeleted) {
-        let e = toBeDeleted[k]
-        try {
-          await(em.deleteException(e.eid))
-        } catch (err) {
-          log.error("Error when delete exception", err);
-        }
+    for (let k in toBeDeleted) {
+      let e = toBeDeleted[k]
+      try {
+        await em.deleteException(e.eid);
+      } catch (err) {
+        log.error("Error when delete exception", err);
       }
-    })()
+    }
   }
 
-  cleanInvalidMACAddress() {
-    return async(() => {
-      const macs = await (hostTool.getAllMACs())
-      const invalidMACs = macs.filter((m) => {
-        return m.match(/[a-f]+/) != null
-      })
-      invalidMACs.forEach((m) => {
-        await (hostTool.deleteMac(m))
-      })
-    })()
+  async cleanInvalidMACAddress() {
+    const macs = await hostTool.getAllMACs();
+    const invalidMACs = macs.filter((m) => {
+      return m.match(/[a-f]+/) != null
+    })
+    return Promise.all(invalidMACs.map((m) => {
+      hostTool.deleteMac(m);
```

Comment:
should use async keyword and return async function.

async (m) => hostTool.deleteMac(m)